development/kops/run_sonobuoy.sh

```diff
@@ -58,6 +58,6 @@ mv $results "./${KOPS_CLUSTER_NAME}/$results"
 results="./${KOPS_CLUSTER_NAME}/$results"
 mkdir ./${KOPS_CLUSTER_NAME}/results
 tar xzf $results -C ./${KOPS_CLUSTER_NAME}/results
-cp ./${KOPS_CLUSTER_NAME}/results/plugins/e2e/results/global/junit_01.xml .
+cp ./${KOPS_CLUSTER_NAME}/results/plugins/e2e/results/global/junit_01.xml /logs/artifacts
```

Comment:
We would want to figure out adding a Spyglass lens here to display the junit output

Reply:
I think, we already have this,
https://github.com/aws/eks-distro-build-tooling/blob/4ad4032bd9fd457853a1d8af7788a[…]harts/stable/prow-control-plane/templates/config-ConfigMap.yaml

Reply:
https://github.com/aws/eks-distro-build-tooling/blob/4ad4032bd9fd457853a1d8af7788a1d36de9dc3c/helm-charts/stable/prow-control-plane/templates/config-ConfigMap.yaml#L59